README.md

```diff
@@ -64,16 +64,16 @@ if __name__ == '__main__':
     root = logging.getLogger()
     root.setLevel(logging.INFO)
     parser = ArgumentParser()
-    parser.add_argument('--steps', type=int)
+    parser.add_argument('--epochs', type=int)
     parser.add_argument('--width', type=float)
     parser.add_argument('--height', type=float)
     args, _ = parser.parse_known_args()
     report = Reporter()
-    for step in range(args.steps):
```

Comment:
Please revert changes from here on until the end of the loop. `dummy_score` depends on `step`, which starts from 0, and then we report `epoch=step + 1`.

Reply:
Done